flink-staging/flink-gelly/src/test/java/org/apache/flink/graph/test/GatherSumApplyITCase.java

```diff
@@ -86,26 +72,35 @@ public void testConnectedComponents() throws Exception {
 
 	@Test
 	public void testSingleSourceShortestPaths() throws Exception {
-		GSASingleSourceShortestPaths.main(new String[]{"1", edgesPath, resultPath, "16"});
-		expectedResult = "1 0.0\n" +
-				"2 12.0\n" +
-				"3 13.0\n" +
-				"4 47.0\n" +
-				"5 48.0\n" +
-				"6 Infinity\n" +
-				"7 Infinity\n";
+		final ExecutionEnvironment env = ExecutionEnvironment.getExecutionEnvironment();
+
+		Graph<Long, Double, Double> inputGraph = Graph.fromDataSet(
+				SingleSourceShortestPathsData.getDefaultEdgeDataSet(env),
+				new InitMapperSSSP(), env);
+
+        List<Vertex<Long, Double>> result = inputGraph.run(new GSASingleSourceShortestPaths<Long>(1l, 16))
+        		.getVertices().collect();
```

Comment:
Is there a specific reason for which we decided to use `collect()` in the tests? It does not seem to be a consistency issue. The other tests (in flink) are still using  `compareResultsByLinesInMemory()`. Do we gain anything?